fetch.bs

```diff
@@ -1150,6 +1209,72 @@ null), run these steps. <var>processBody</var> must be an algorithm accepting a
  <var>reader</var>.
 </ol>
 
+<p>To <dfn export for=response>incrementally read</dfn> a <a for=/>response</a> <var>response</var>,
+given an algorithm <var>processBodyChunk</var>, an algorithm <var>processEndOfBody</var>,
+an algorithm <var>processBodyError</var>, and an optional null,
+<a for=/>parallel queue</a>, or <a for=/>global object</a> <var>taskDestination</var> (default
+null), run these steps.
+<var>processBodyChunk</var> must be an algorithm accepting a <a for=/>byte sequence</a>.
+<var>processEndOfBody</var> must be an algorithm accepting no arguments. <var>processBodyError</var>
+must be an algorithm accepting an exception.
+<ol>
+ <li><p>Let <var>processAndReportEndOfBody</var> be an action the performs the following steps:
```

Comment:
```suggestion
 <li>
  <p>Let <var>processAndReportEndOfBody</var> be the following steps:
```
(and similar below)

I don't think we need "an action" anymore for identifying algorithms.